integration-tests/mtls-certificates/src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+quarkus.http.ssl.certificate.key-store-file=server-keystore.jks
+quarkus.http.ssl.certificate.key-store-password=password
+quarkus.http.ssl.certificate.key-store-key-alias=server
+quarkus.http.ssl.certificate.key-store-key-password=serverpw
+quarkus.http.ssl.certificate.trust-store-file=server-truststore.jks
+quarkus.http.ssl.certificate.trust-store-password=password
+quarkus.http.ssl.client-auth=REQUIRED
+quarkus.http.auth.certificate-role-properties=role-mappings.txt
```

Comment:
This is a good question. Woudln't we want to support both styles of configuration: embedded in main `application.properties` and in an extra `.properties` file, similarly to https://quarkus.io/guides/security-properties ?

Reply:
@michalvavrik @Ladicek Good points, thanks, the file is supported because this is what the customer already has, so asking them to move them to the explicit declarations in application.properties would not be ideal, but yeah, worth supporting too, I'll have a look

Reply:
@sberyozkin is there a difference for customers between `quarkus.http.auth.certificate-role-properties=role-mappings.txt` and `quarkus.config.locations=path-to-file-role-mappings.txt` ? I didn't try that as I'm busy today, but I think it is worth to consider that there can be no actual difference (please try it to determine if that is a valid point). Also if customers prefer colon format, they can use YAML extension. Thanks

Reply:
@michalvavrik Not sure `quarkus.config.locations` works well, it is creating some kind of config indirection, IMHO having a dedicated property is not bad; `:` is fixed by Ladislav, `=` is suggested now

Reply:
@sberyozkin hey difference would be config property names prefix. Also with `io.smallrye.config.source.file.FileSystemConfigSource` you could 
 `io.smallrye.config.source.file.FileSystemConfigSource#getValue` have a full control in dropping config prefix (`quarkus.http.auth.certificate-role-properties.` in overridden method, adding prefix to keys by overriding `getPropertyNames`) and user's would be able to leverage SR Config features. 

I'm fine if you just add support for config based roles.

Reply:
IMO keeping what you have is the simplest.

Reply:
@michalvavrik, I've been thinking about it, let me clarify something, so this PR is about mapping certificate CN names (which is only one, though primary certificate attribute) to roles, so with

`quarkus.http.auth.certificate-role.client=user,admin`

it would introduce authentication specific (MTLS) specific property (`certificate-role`) into a generic HTTP security policy configuration, and I'm not sure now we should do it.

The other thing is, something I've found out with #37152 (and I've now removed the related code), CN can be complex, they may have tricky characters, so ended up doing something like:
```
quarkus.oidc.certificate-chain.root-aliases.myCA=Root CA
quarkus.oidc.certificate-chain.endpoint-authorities.myCA=www.quarkustest.com
```
where I was mapping the CA name to some alias, note, I had to set `Root CA` as a map value, as there were issues either with `Root CA` or `www.quarkustest.com` (this is the subject cert CN) set as map keys at the config level.

I see, at #23683 I was also thinking about some maps at the config level, but after #37152 I decided to leave it, if necessary till later.

My proposal is this: since we agree that supporting file-based mapping is fine and meets a concrete requirement, lets have this PR be specific about that requirement only. And I can open an enhancement request to complement it with the mapping at the config level, and I'll copy your proposal there, and we'll keep it recorded, how does it sound ?

Reply:
Hi @michalvavrik, I'll create an enhancement request once this PR gets one more approval and I can merge it